src/bicep/mlz.bicep

```diff
@@ -1048,6 +1048,10 @@ output hub object = {
   networkSecurityGroupResourceId: hubNetwork.outputs.networkSecurityGroupResourceId
 }
 
+output deployDefender bool = deployDefender
```

Comment:
We are adding additional outputs to the MLZ baseline; this is to incorporate adding in outputs for the deploymentVariables.json for Tier3. 

While this isn't a big change, we should probably ensure that the outputs listed in the README.md of the MissionLZ bicep code include the outputs linked [in the outputs documentation](https://github.com/Azure/missionlz/tree/main/src/bicep#outputs) 

Please add to the list: 

deployDefender.value
emailSecurityContact.value